scripts/check-banned-deps.ts

```
@@ -0,0 +1,428 @@
+    .filter((p) => existsSync(p));
+  while (stack.length > 0) {
+    const dir = stack.shift()!;
+    let entries: string[];
+    try {
+      entries = readdirSync(dir);
+    } catch {
+      continue;
+    }
+    for (const entry of entries) {
+      const full = join(dir, entry);
+      let stat;
```

Biome (2.4.14)

[error] This variable implicitly has the any type.

(lint/suspicious/noImplicitAnyLet)